src/coreclr/jit/emitarm64.cpp

```diff
@@ -18844,8 +18938,17 @@ void emitter::emitDispInsHelp(
         case IF_SVE_CX_4A_A: // ........xx.mmmmm ...gggnnnnn.DDDD -- SVE integer compare vectors
             emitDispPredicateReg(id->idReg1(), PREDICATE_SIZED, id->idInsOpt(), true); // DDDD
             emitDispPredicateReg(id->idReg2(), PREDICATE_ZERO, id->idInsOpt(), true);  // ggg
-            emitDispSveReg(id->idReg3(), id->idInsOpt(), true);                        // mmmmm
-            emitDispSveReg(id->idReg4(), INS_OPTS_SCALABLE_D, false);                  // nnnnn
+            emitDispSveReg(id->idReg3(), id->idInsOpt(), true);                        // nnnnn
+            emitDispSveReg(id->idReg4(), INS_OPTS_SCALABLE_D, false);                  // mmmmm
+            break;
+
+        // <Pd>.<T>, <Pg>/Z, <Zn>.<T>, #<imm>
+        case IF_SVE_CY_3A: // ........xx.iiiii ...gggnnnnn.DDDD -- SVE integer compare with signed immediate
+        case IF_SVE_CY_3B: // ........xx.iiiii ii.gggnnnnn.DDDD -- SVE integer compare with unsigned immediate
+            emitDispPredicateReg(id->idReg1(), PREDICATE_SIZED, id->idInsOpt(), true); // DDDD
+            emitDispPredicateReg(id->idReg2(), PREDICATE_ZERO, id->idInsOpt(), true);  // ggg
```

Comment:
Here we should be using `insGetPredicateType(fmt)` instead of `PREDICATE_ZERO`.

But... on the previous line we would still need to do `PREDICATE_SIZED`.
We could extend `insGetPredicateType()` to have a an extra arg (reg position), which is ignored for most instructions.

Alternatively, we keep this PR as it is.

Reply:
> Alternatively, we keep this PR as it is.

I agree. But we should have a follow up PR that calls `insGetPredicateType()` instead of hard-coding even at other places.

Reply:
> I agree. But we should have a follow up PR that calls `insGetPredicateType()` instead of hard-coding even at other places.

Done: https://github.com/dotnet/runtime/pull/97142